# watt2021_yap

## Contents

1. runstar_mouse.sh is the code used to map RNA-seq reads to the genome with STAR

2. fcount.sh is the code used to count reds mapped to exons to generate gene level counts.

3. starlog.sh is a script to summarise/tabulate star log information

4. bwa_mouse_rRNA.sh is a script to count rRNA reads to see if the mRNA enrichment worked well.

5. edgeR.Rscript is the R script used to run the differential expression testing.

6. deg_compare.Rmd is an R script used to compare different yap knock down data

7. yapkd_mitch.Rmd is the R script used to run the mitch analysis

8. bdg.Rmd is the script used to create the bedgraph and to create coverage plots at the specific regions.
